frontend-react/src/styles/_components.scss

```diff
@@ -115,4 +115,3 @@ hr {
         margin-top: 40px;
     }
 }
-
```

Comment:
Undo this change to prevent chromatic from running.

Reply:
Thank you @jpandersen87! Fixed. (I'm at a bit of a loss how this made it into the merge commit to begin with...)